lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -234,9 +269,16 @@ def main():
         containers=dict(required=False, type='list'),
         network_mode=dict(required=False, default='bridge', choices=['bridge', 'host', 'none', 'awsvpc'], type='str'),
         task_role_arn=dict(required=False, default='', type='str'),
-        volumes=dict(required=False, type='list')))
+        volumes=dict(required=False, type='list'),
+        launch_type=dict(required=False, choices=['EC2', 'FARGATE'], default='EC2'),
+        cpu=dict(required=False, type='str'),
```

Comment:
This can just be `cpu=dict()` as both `required=False` and `type='str'` are the default.